lib/gollum/app.rb

```diff
@@ -15,14 +15,14 @@
 require 'gollum/assets'
 require 'gollum/views/helpers'
 require 'gollum/views/helpers/locale_helpers'
+require 'gollum/views/template_cascade'
 require 'gollum/views/layout'
 require 'gollum/views/editable'
 require 'gollum/views/has_page'
 require 'gollum/views/has_user_icons'
 require 'gollum/views/has_math'
 require 'gollum/views/pagination'
 require 'gollum/views/rss.rb'
-require 'gollum/views/template_cascade'
```

Comment:
Is the order of these requires important, or is it just an incidental change?

If it's important, it might be worthwhile to add an inline comment explaining why, so the future versions of us don't get caught by the order-related issues again.

Reply:
Yes, order is important now because our layout now includes `TemplateCascade` directly, instead of the layout being extended dynamically later on in this file. I'll add a note!